### DISCUSSION TITLE
Problem with the gh-page deploy Media files.

### DISCUSSION THREAD

**Discussion by SohamTilekar** [mkdocs/mkdocs#3573] [Q&A] [answered]
I have a Folder name media Which contain some Images like logo.png, favicon.png, etc.
Those Work Fine With the local path But When I Use the action to deploy then on the THe Website Do not Contain those Images.

This Is Because of the Wrong Path to the image.
[Wrong Path](https://sohamtilekar.github.io/vidiopy/media/vidiopy_logo.png) : -https://sohamtilekar.github.io/vidiopy/media/vidiopy_logo.png
[Right Path](https://github.com/SohamTilekar/vidiopy/blob/gh-pages/media/vidiopy_logo.png?raw=true) : - https://github.com/SohamTilekar/vidiopy/blob/gh-pages/media/vidiopy_logo.png?raw=true

Bellow is the Screen Shot of the image.
No Logo Image: -![No Image](https://github.com/mkdocs/mkdocs/assets/152298897/3af87331-f610-4d72-a3d2-64b89866f74d)
No favicon Image: -![No Favicon](https://github.com/mkdocs/mkdocs/assets/152298897/8d812a8d-9a6e-495e-bc02-b94fd22f380a)

**Comment by oprypin** [comment]
https://media.githubusercontent.com/media/SohamTilekar/vidiopy/gh-pages/media/vidiopy_logo.png
-instead of being an image, it has the following content:
```
version https://git-lfs.github.com/spec/v1
oid sha256:4ce94e2030728b351267f7128f378784a376db655bde4c0d60456125ab570636
size 516991
```
You should try to not store it with Git LFS.

**Reply by SohamTilekar** [reply]
Remove it From the Git LFS but Still Same Issue.

**Reply by oprypin** [reply, marked as answer]
@SohamTilekar No, it works now, just clear your cache with Ctrl+F5

**Reply by SohamTilekar** [reply]
But why there is a problem with the git LFS.

**Reply by oprypin** [reply]
That can be a valid question. MkDocs is older than Git LFS, maybe nobody even thought about it, or it's a rare use case

**Reply by SohamTilekar** [reply]
Should I start a feature request or not because handling the heavy media file in rep is not suggested. And media files are used to much.
